CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 🚀 Check out the guidelines [here](https://github.com/xcodeswift/contributors/blob/master/CHANGELOG_GUIDELINES.md)
 
+## master
```

Comment:
These changes are not necessary. We'll tag this PR with the proper milestone and we'll update the Changelog accordingly when we make the next release.

Reply:
Reverted👌